sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -926,6 +1176,25 @@ public partial class RecognizePiiEntitiesResultCollection : System.Collections.O
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RelativeTo : System.IEquatable<Azure.AI.TextAnalytics.RelativeTo>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
```

Comment:
I am curious, what are these things?

Reply:
I had the same question. These are auto-generated, private, and show up in other places that pre-date me (including in other libraries too). 

I asked for more info. From Heath:
> Those are just artifacts from the metadata reader we get from structs.